apkid/rules/dex/obfuscators.yara

```diff
@@ -241,3 +242,23 @@ rule aamo_str_enc : obfuscator
   condition:
     1 of ($opcodes*) and all of ($a, $b)
 }
+
+rule appsuit : obfuscator
+{
+    meta:
+        description = "AppSuit"
+        url         = "http://www.stealien.com/appsuit.html"
+        example     = "b99bafbbd5288ac93647d22f1c5b1863c96f581ae7a19fdc0e84bff4c2141328"
```

Comment:
Could you use `sample` for this? Some used sample, some used example, so I just standardized on `sample`. This is used in automated testing.

Reply:
Done!